SimplenoteShare/Base.lproj/MainInterface.storyboard

```diff
@@ -1,21 +1,25 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can we revert the changes on this file? (since it's not really needed for this PR!)